<launch>

  <!-- bagfile params -->
  <arg name="cloud_reg"        default="$(find ramon_slam3d)/pcd/rosbot/turtlebot3_world/2020-08-16-12-15-37/cloud1.pcd"/>
  <arg name="cloud_prev"       default="$(find ramon_slam3d)/pcd/rosbot/turtlebot3_world/2020-08-16-12-15-37/cloud20.pcd"/>

  <node name="pcl_test"	
        pkg="ramon_slam3d"  
        type="pcl_test" 
        output="screen"
        clear_params="true">
    <param name="cloud_reg" value="$(arg cloud_reg)"/>
    <param name="cloud_prev" value="$(arg cloud_prev)"/>
  </node>

</launch>